### PR TITLE
Add all 3 reporting options to SECURITY.md

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -17,9 +17,17 @@ This is a shared security policy for the CosmWasm stack, including the following
...
+| Eligable for bounty | yes                                                   | no                                                                                                                                                                                                                                                                                                  | no                                                                                                                                                                                                                                |
```
Comment (chipshort) [inline SECURITY.md:25]:
```suggestion
| Eligible for bounty | yes                                                   | no                                                                                                                                                                                                                                                                                                  | no                                                                                                                                                                                                                                |
```

**Review by chipshort — APPROVED** [review]
LGTM

File: SECURITY.md (hunk 1 above)
Comment (webmaster128) [reply SECURITY.md:25]: Thanks – I always mess up this word

Comment (webmaster128) [reply SECURITY.md:25]: Now let's see the autofixer in action for re-formatting the table ...

Comment (webmaster128) [reply SECURITY.md]: Nevermind, no change needed

**Pull request merged**